src/components/editors/remote/remote_editors.gd

```diff
@@ -114,8 +96,8 @@ func install_zip(zip_abs_path, root_unzip_folder_name, possible_editor_name, on_
 		var editor_install = _editor_install_scene.instantiate()
 		add_child(editor_install)
 		editor_install.init(possible_editor_name, zip_content_dir)
-		editor_install.installed.connect(func(name, exec_path):
-			installed.emit(name, ProjectSettings.globalize_path(exec_path))
+		editor_install.installed.connect(func(p_name, exec_path):
```

Comment:
`name` is a reserved variable from `Node`, therefore I changed it to avoid conflicts.